src/challenges/defi-challenges/simple-lending.mdx

```diff
@@ -0,0 +1,221 @@
+---
```

Comment:
Do we need to rebase on main since the lending challenge already exists? Or has it been modified here

Reply:
My fork got a bit messy :sweat_smile:
It should be fixed now.